proxy/secret.go

```diff
@@ -223,6 +223,9 @@ func CreateSecretToken(gateway string, secret schema.Secret, tlsInsecure bool, t
 	case http.StatusUnauthorized:
 		output += fmt.Sprintln("unauthorized access, run \"faas-cli login\" to setup authentication for this server")
 
+	case http.StatusConflict:
+		output += fmt.Sprintln("a secret with the same name already exists, run \"faas-cli secret update\" instead")
```

Comment:
I wonder whether we should keep the message objective. Updating a pre-existing secret could have undesired effects in some scenarios.  The user may instead use a different name.

Reply:
What would you suggest should happen in this case?

Reply:
That the user should find their own path since they know better the context of their environment than the CLI does.

Informing the user that the secret already exists helps them understand the cause of the failure without leading them towards one particular action.

Either that or add “doing so may have undesired effects on existing functions that are already using this secret”

Reply:
I agree with @rgee0 , there are two equally valid options for the use (1) use a different name or (2) update the existing secret.  Why should the CLI direct you to the second option? I think it would be good enough to just say `a secret with the same name already exists`

Reply:
I'm good with what @rgee0 and @LucasRoesler said.

@bmcstudio thank you for handling the specific error, could you return the message such as mentioned above?

We'll let the user decide what to do when they have a secret that already exists rather than directing them to the *update* command.

Reply:
@alexellis @LucasRoesler @rgee0 thank you, I've updated the error message.